spread-tests/main/mount-ns-layout/expected.json

```diff
@@ -0,0 +1,662 @@
+[
+  {
+    "fs_type": "squashfs",
+    "mount_opts": "rw,relatime",
+    "mount_point": "/",
+    "mount_src": "/dev/loop0",
+    "opt_fields": [
+      "master:renumbered/0"
+    ],
+    "root_dir": "/"
+  },
+  {
+    "fs_type": "devtmpfs",
+    "mount_opts": "rw,nosuid,relatime",
+    "mount_point": "/dev",
+    "mount_src": "udev",
+    "opt_fields": [
+      "master:renumbered/1"
+    ],
+    "root_dir": "/"
+  },
+  {
+    "fs_type": "devpts",
+    "mount_opts": "rw,relatime",
+    "mount_point": "/dev/ptmx",
+    "mount_src": "devpts",
+    "opt_fields": [],
+    "root_dir": "/ptmx"
+  },
+  {
+    "fs_type": "devpts",
+    "mount_opts": "rw,relatime",
+    "mount_point": "/dev/pts",
+    "mount_src": "devpts",
+    "opt_fields": [],
+    "root_dir": "/"
+  },
+  {
+    "fs_type": "devpts",
+    "mount_opts": "rw,nosuid,noexec,relatime",
+    "mount_point": "/dev/pts",
+    "mount_src": "devpts",
+    "opt_fields": [
+      "master:renumbered/2"
+    ],
+    "root_dir": "/"
+  },
+  {
+    "fs_type": "tmpfs",
+    "mount_opts": "rw,nosuid,nodev",
+    "mount_point": "/dev/shm",
+    "mount_src": "tmpfs",
+    "opt_fields": [
+      "master:renumbered/3"
+    ],
+    "root_dir": "/"
+  },
+  {
+    "fs_type": "ext4",
+    "mount_opts": "rw,noatime",
+    "mount_point": "/etc",
+    "mount_src": "/dev/sda",
```

Comment:
Isn't the `/dev/sda` portion going to be brittle? For example, if testing in QEMU using a virtio, I suspect this will be `/dev/vda`.